### PR TITLE
Modals: Create Instance and Exit

### PR DESCRIPTION
**Pull request by cheina97**
Components included:

- Modal Create Instance
- Modal Exit

### CODE REVIEW

File: frontend/src/components/workspaces/Modal/ModalAlert/ModalAlert.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { Modal, Alert } from 'antd';
+
+export interface IModalAlertProps {
+  headTitle: string;
+  showModal: boolean;
+  alertMessage: string;
+  alertDescription: string;
+  alertType: 'info' | 'success' | 'error' | 'warning' | undefined;
```
Comment (GabriFila) [inline frontend/src/components/workspaces/Modal/ModalAlert/ModalAlert.tsx:9]: I tried the `undefined` value and the app crashes. I don't know if it necessary, if it is you need to handle it properly

Comment (cheina97) [reply frontend/src/components/workspaces/Modal/ModalAlert/ModalAlert.tsx:9]: I removed it

**Review by GabriFila — APPROVED** [review]

**Review by QcFe — APPROVED** [review]
lgtm

**Pull request merged**